docker/slurm/docker-compose.yml

```diff
@@ -0,0 +1,74 @@
+version: "3.8"
+
+services:
+
+  mysql:
+    image: mysql:5.7
+    hostname: mysql
+    container_name: mysql
+    environment:
+      MYSQL_RANDOM_ROOT_PASSWORD: "yes"
+      MYSQL_DATABASE: slurm_acct_db
+      MYSQL_USER: slurm
+      MYSQL_PASSWORD: password
+    volumes:
+      - var_lib_mysql:/var/lib/mysql
+
+  slurmdbd:
+    image: slurm-docker-cluster
+    build:
+      context: ../..
+      dockerfile: ./docker/slurm/Dockerfile
+    command: [ "slurmdbd" ]
+    container_name: slurmdbd
+    hostname: slurmdbd
+    volumes:
+      - etc_munge:/etc/munge
+      - etc_slurm:/etc/slurm
+      - var_log_slurm:/var/log/slurm
+    expose:
+      - "6819"
+    depends_on:
+      - mysql
+
+  slurmctld:
+    image: slurm-docker-cluster
```

Comment:
Please, replace all 

> image: slurm-docker-cluster

occurrences with `build` section:
```
build:
  context: ../..
  dockerfile: ./docker/slurm/Dockerfile
```

It will allow us to ensure, that custom image buily from `slurm/Dockerfile` is used

Reply:
In this case, docker will create 3 identical images instead of one. Is this OK?

Reply:
Sorry, I missed the fact, that the `build` stage is specified for the first service, so the following ones are going to use the same image.
Can you, please, change the image name for something more specific, like  `slurm-base-grid-engine-api`